CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Remove trailing slash from endpoint url by [@Procta].
 - Added local cache to getResolutions by [@jpastoor].
 - Renamed Api::api() parameter $return_as_json to $return_as_array by [@jpastoor].
+- Rename createRemotelink -> createRemoteLink [@glensc].
```

Comment:
Please change to:

```
Renamed `Api::createRemotelink` into `Api::createRemoteLink` by [@glensc].
```

Also you should map your GitHub username to GitHub user profile page at the bottom of the file to make your username clickable.

Reply:
updated. the link was added already by my previous contribution